web/src/components/review-form.tsx

```diff
@@ -145,16 +149,23 @@ export default function SwipeReview(props: Props) {
 
   return (
     <form id="review-form" onSubmit={onSubmit}>
-      <Prompt
-        when={reviewedSentencesCount > 0}
-        message="Reviewed sentences not submitted, are sure?"
-      />
-
+      <Localized id="sc-review-title" attrs={{ message: true }}>
+        <Prompt
+          when={reviewedSentencesCount > 0}
+          message="Reviewed sentences not submitted, are sure?"
+        />
+      </Localized>
+
+      {/* TODO & TO CHECK : Do these messages need translation?  */}
       {message && <p>{message}</p>}
 
       <p className="small">
-        Swipe right to approve the sentence. Swipe left to reject it. Swipe up to skip it.{' '}
-        <strong>Do not forget to submit your review!</strong>
+        <Localized id="sc-review-form-usage-1">
+          Swipe right to approve the sentence. Swipe left to reject it. Swipe up to skip it.
+        </Localized>{' '}
+        <Localized id="sc-review-form-usage-2">
+          <strong>Do not forget to submit your review!</strong>
+        </Localized>
```

Comment:
I was having trouble with "strong" markup being stripped off. Now I found this: https://github.com/projectfluent/fluent.js/wiki/React-Overlays

It is now working like the following:

`<Localized id="sc-review-form-usage" elems={{ strong: <strong></strong> }}>`

This might also be a case in others...

Reply:
Tested OK.